packages/globals/src/keyboard-layout.ts

```diff
@@ -99,11 +99,60 @@ export type KeyCode =
   | "Slash"
   | "Backquote"
   | "BracketLeft"
-  | "Backslack"
+  | "Backslash"
   | "BracketRight"
   | "Quote";
 
+export const FunctionalKeys: KeyCode[] = [
```

Comment:
Let's just go ahead and import all of the different key types from MDN https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values and name them after what is found there.

This way, we will be sure and have an exhaustive list, and the category of key will be aligned with what is found there.

Reply:
Done

Reply:
Does it make sense to make each key type in its own array so that we can filter accordingly? I'm not sure what "FunctionalKeys" means or if maybe it should be a combination of smaller categories of key code

Reply:
There are two different properties of a `KeyboardEvent`: `key` and `code`. Here for `FunctionalKeys` and `KeyCode` we described codes and real keys defined in layouts (like for example in `defaultKeyMap` for US layout). But for functional keys `key` and `code` almost the same and I need them to filter as non-printable characters.

In reality it's more complicated, like `backspace` removes the last character, but we don't have such functionality.

So what about `KeyValues` they are described `key` property, but I don't know what `code` they should have. And I think I got your point. We don't filter all these key values as non-printable characters.